### DISCUSSION TITLE
Paperless LXC - Update - No installation found

### DISCUSSION THREAD

**Discussion by MedusaLeiche** [tteck/Proxmox#3556] [Help Needed]
I have a Paperless NGX installation running. When i try to update it using the bash in the LXC console, I get the answer: 
No Paperless-ngx Installation Found!
And when I type "update" in the console I geht the answer "-bash: update: command not found"
Whats my problem? 
![Screenshot 2024-08-20 162614](https://github.com/user-attachments/assets/259e6e6b-c66a-4e73-bdab-5b8f51db9940)

**Comment by MickLesk** [comment]
is your installation in /opt?

Like this:
root@paperless-ngx:/opt/paperless
